src/admin/class-wordlift-admin-entity-list.php

```diff
@@ -80,4 +80,73 @@ public function render_custom_columns( $column, $entity_id ) {
 		}
 		
 	}
+	
+	/**
+	 * Add 4W select box before the 'Filter' button.
+	 *
+	 * @since 3.3.0
+	 */
+	public function add_4W_filter() {
+		global $typenow;
+		
+		// Only show on entity list page
+		if( $typenow !== Wordlift_Entity_Service::TYPE_NAME ){
```

Comment:
There is method in Entity Service which checks whether a type is an entity. Better use that.